compilation/platforms/truffle.go

```diff
@@ -65,6 +65,8 @@ func (s *TruffleCompilationConfig) Compile() ([]types.Compilation, string, error
 	// Create a compilation unit out of this.
 	compilation := types.NewCompilation()
 
+	// TODO: Parse a source list!
```

Comment:
Is this meant to be left as a TODO ?

Reply:
This was removed in https://github.com/crytic/medusa/pull/111/commits/a83955a3df63d4b15938c029150a54e05d72ec83